labs/firepower-restapi-112/1.md

```diff
@@ -1,23 +1,20 @@
 # FMC - REST - API - 112
 
-## Lab Objective
+## Objectives
 
-Basic understanding of Firepower Management Center (FMC) REST based APIs CRUD operations   
+This lab teaches you the basic understanding of Firepower Management Center (FMC) REST based APIs CRUD operations.   
 
-Completion time: 25 minutes
 
 ## Prerequisites
-Basic understanding of REST principles, firewalls
+Basic understanding of REST principles and firewalls.
 
 
-## Learning Objective
-After completing this module you will have a basic understanding of FMC REST APIs CRUD operations
+## Learning objective
+After completing this module you will have a basic understanding of FMC REST APIs CRUD operations.
 
 
 ## Read operation GET
-If you are using DevNet sandbox  https://fmcrestapisandbox.cisco.com/api/api-explorer/
-
-For the CLUS 17 workshop session the username and password is provided to you. These credentials are only valid for the CLUS17 session.
+If you are using the [DevNet Sandbox] (https://fmcrestapisandbox.cisco.com/api/api-explorer/), for this workshop session the username and password is provided to you. These credentials are only valid for this session.
```

Comment:
Having a space after the square bracket `]` means the link will not render correctly. I wonder also why this link is the fmcrestapisandbox, but in the prior PR the link was https://cisco-tid.cisco.com/api/api-explorer/?